generator/internal/language/rust.go

```diff
@@ -1046,12 +1047,30 @@ func (c *RustCodec) Validate(api *api.API) error {
 
 // RustContext contains Rust specific data that can be referenced in templates.
 type RustContext struct {
-	HasFeatures bool
-	Features    []string
+	HasFeatures       bool
+	Features          []string
+	RemoveTypeWrapper mustache.LambdaFunc
+}
+
+// removeTypeWrapper removes a container type around another. For example it
+// would transform `Vec<String>` to `String`.
+func removeTypeWrapper(text string, render mustache.RenderFunc) (string, error) {
```

Comment:
Lmk if you don't like this approach. This seemed like an easier way to modify information the codec already spits out.

Reply:
This also might be a way to register some global cross language modifiers in the future...

Reply:
See below, but maybe you have some ideas for future changes?